x-pack/plugins/uptime/server/rest_api/uptime_route_wrapper.ts

```diff
@@ -10,13 +10,16 @@ import { savedObjectsAdapter } from '../lib/saved_objects';
 export const uptimeRouteWrapper: UMKibanaRouteWrapper = uptimeRoute => {
   return {
     ...uptimeRoute,
+    options: {
+      tags: ['access:uptime-read', ...(uptimeRoute.options?.tags ?? [])],
```

Comment:
Since we only have two options read/write, maybe instead of adding arbitrary tags we can just add a `accessWrite` option or similar that adds `access:uptime-write`.